packages/crypto/lib/src/crypto/zip32.rs

```diff
@@ -124,11 +124,11 @@ mod tests {
             .expect("Deriving from ExtendedKeys should not fail");
 
```

Comment:
Weird, I'm now getting a build warning on this file, though not sure why it's happening on this branch:
```
warning: field `0` is never read
  --> src/crypto/zip32.rs:15:27
   |
15 | pub struct ExtSpendingKey(pub(crate) ExtendedSpendingKey);
   |            -------------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |            |
   |            field in this struct
```
Not sure that we're using that type any more. Not a big deal on this PR, just noting!